client/stylesheets/_responsive.scss

```diff
@@ -695,49 +530,21 @@
 }
 
 @media (max-width: 321px) {
-
     .homeCardLi {
         width: 110px;
     }
-
-    .homeCardImg {
-        height: 110px !important;
-    }
-
-    .searchResultsCardName p {
-        font-size: 0.2rem !important;
-        margin: 0 !important;
-    }
-
-    .searchResultsCardImg {
-        height: 100px !important;
-    }
-
-    .sortClick ul {
-        margin-right: -18px !important;
-    }
 }
 
-@media (max-width: 314px) {
-    .searchResultsCardName {
-        display: none !important;
+@media (max-width: 300px) {
+    .helpImgUl > li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `helpImgUl` should be written in lowercase with hyphens